comps/guardrails/factuality/predictionguard/requirements.txt

```diff
@@ -0,0 +1,13 @@
+aiohttp
+docarray
+fastapi
+huggingface_hub
+opentelemetry-api
+opentelemetry-exporter-otlp
+opentelemetry-sdk
+Pillow
+predictionguard==2.2.1
```

Comment:
why use `predictionguard==2.2.1`? this could cause unmatched dependency version when other dependencies update.